Casks/p/plotdigitizer.rb

```diff
@@ -1,12 +1,17 @@
 cask "plotdigitizer" do
-  version "2.6.11"
-  sha256 "527f4eaf178b358e0d4de8f99782412665cc15601ff673d6a58a490787288925"
+  version "2.6.11b"
+  sha256 "55bd67c08b169ad9122a1f0e55d80d4e5e8da3a7787e1e0595a0bd27f9dadaad"
 
-  url "https://downloads.sourceforge.net/plotdigitizer/#{version}/PlotDigitizer_MacOS.dmg"
+  url "https://downloads.sourceforge.net/plotdigitizer/plotdigitizer/#{version}/PlotDigitizer_MacOS.dmg"
   name "Plot Digitizer"
   desc "Digitize scanned plots of functional data"
   homepage "https://plotdigitizer.sourceforge.net/"
 
+  livecheck do
+    url "https://sourceforge.net/projects/plotdigitizer/rss?path=/plotdigitizer"
+    regex(%r{url=.*?/v?(\d+(?:\.\d+)+[a-z]?)/PlotDigitizer[._-]MacOS\.dmg}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/v?(\d+(?:\.\d+)+[a-z]?)/PlotDigitizer(?:[._-]v?\d+(?:\.\d+)+)?[._-]MacOSX?\.dmg}i)
```

Previous releases included the version in the filename and used a `MacOSX` suffix, so we should probably handle that format as well.